windows/tools/nocli/Main.cxx

```diff
@@ -71,13 +71,15 @@ enum class LogInstanceId {
 int
 main(int argc, char* argv[])
 try {
+    static plog::DebugOutputAppender<plog::TxtFormatter> debugAppender{};
     static plog::ColorConsoleAppender<plog::MessageOnlyFormatter> colorConsoleAppender{};
-    plog::init<static_cast<int>(LogInstanceId::Console)>(plog::info, &colorConsoleAppender);
-
     static plog::RollingFileAppender<plog::TxtFormatter> rollingFileAppender(logging::GetLogName("nocli").c_str());
-    plog::init<static_cast<int>(LogInstanceId::File)>(plog::verbose, &rollingFileAppender);
 
-    plog::init(plog::verbose).addAppender(plog::get<static_cast<int>(LogInstanceId::Console)>()).addAppender(plog::get<static_cast<int>(LogInstanceId::File)>());
+    plog::init<static_cast<int>(LogInstanceId::Console)>(plog::verbose, &colorConsoleAppender);
+    plog::init<static_cast<int>(LogInstanceId::File)>(plog::verbose, &rollingFileAppender);

```

Comment:
What is the reason for adding these two additional `plog::init`s?

Reply:
They were there previously, just moved to be declared together with other statics. The one that was added is the `DebugOutputAppender`. That will forward log data to the a debugger window, if attached. It's really useful when you're using the debugger and want to see things in realtime as opposed to setting up ETW logging separately.